survey_contact_generation/models/res_partner.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2024 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    survey_user_input_id = fields.Many2one(comodel_name="survey.user_input")
```

Comment:
Shouldn't be the reverse direction? Put the generated partner in the survey_input?

Reply:
Already is, but I wan't also to flag those generated partners so I can easily discriminate them for other workflows (look at the tt task for an example)

Reply:
And what about the inverse one2many?

Reply:
I just want to link inputs to partners generated by that input, but the input could have a partner which isn't generated (that's the normal behavior)

Reply:
OK, let's try to make it more self-discovering by name:

```suggestion
    generating_survey_user_input_id = fields.Many2one(comodel_name="survey.user_input")
```

Reply:
Done :+1: